From 24449ea8c7eca61c08b955c95ea7c5295a2a2bde Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Wed, 27 Mar 2013 17:50:08 +0100
Subject: [PATCH] build:

AFAICS, VMIME broke its API between 0.4.x and 0.8.x without updating
soversions, hence no trust that it is correct now. Add some suffix
that is unique within our distro and distinguishes itself from
upstream.
---
 CMakeLists.txt |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: vmime/CMakeLists.txt
===================================================================
--- vmime.orig/CMakeLists.txt
+++ vmime/CMakeLists.txt
@@ -52,7 +52,7 @@ SET(VMIME_VERSION_MICRO  2)
 # . Interfaces added/removed/changed:               CURRENT++, REVISION=0
 # . Interfaces added (upward-compatible changes):   AGE++
 # . Interfaces removed:                             AGE=0
-SET(VMIME_API_VERSION_CURRENT   0)
+SET(VMIME_API_VERSION_CURRENT   2)
 SET(VMIME_API_VERSION_REVISION  0)
 SET(VMIME_API_VERSION_AGE       0)
 
@@ -65,7 +65,7 @@ SET(VMIME_API_VERSION ${VMIME_API_VERSIO
 # VMime Library
 
 # Set base name
-SET(VMIME_LIBRARY_NAME vmime)
+SET(VMIME_LIBRARY_NAME vmime-0.9.2)
 
 # Source files
 FILE(
